<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CryptoPortfolio QT</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableView" name="assetsTable">
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::SolidLine</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>false</bool>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>51</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;b&gt;Quantity:&lt;/b&gt;</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="quantityEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>51</y>
           <width>110</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="currencyComboBox">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>10</y>
           <width>111</width>
           <height>26</height>
          </rect>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>12</y>
           <width>64</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;b&gt;Currency:&lt;/b&gt;</string>
         </property>
        </widget>
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>146</width>
           <height>66</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="addAssetButton">
            <property name="text">
             <string>Add Position</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeAssetButton">
            <property name="text">
             <string>Remove Position</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
